### ISSUE TITLE
GraphQL pokémon sprites returns 404 status

### ISSUE THREAD

**Issue by assiscs2** [PokeAPI/pokedex#146]
I was taking a look in my pokedex project and it seems that something is broken on retrieving the Pokémon sprites via GraphQL.

Here's the projects link: https://pokedex-react-topaz-zeta.vercel.app/

I'm pretty sure its not a problem from my side, since I haven't done any changes in this project for quite some time.

The console returns a 404 error: "Failed to load resource: the server responded with a status of 404 ()"
![Print pokedex](https://user-images.githubusercontent.com/50466377/218560946-2459079d-4b3e-47d1-a5e8-abecd820ec66.png)

**Comment by Naramsim** [comment]
Yes, there's a problem in our GraphQL API. Currently we strip away https://raw.githubusercontent.com/PokeAPI/sprites/master/, so you have to manipulate the link. 

You need to change `/media/` to `https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/`

**Comment by assiscs2** [comment]
@Naramsim, thanks for the reply!

I don't know if I'm doing it wrong, but it's still not working for me.

I have manipulated the link, but I still get the same result, are any of these two images below the correct changes?

![Erro pokedex 1](https://user-images.githubusercontent.com/50466377/218910337-2bc801b9-f944-461c-b8fa-fa3e172c0f20.png)

![Erro pokedex 2](https://user-images.githubusercontent.com/50466377/218910345-446652b1-b7d9-4997-8eaa-f6bc3c398e34.png)
